### DISCUSSION TITLE
Welcome to thunder-client Discussions!

### DISCUSSION THREAD

**Discussion by rangav** [thunderclient/thunder-client-support#498] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by akshita-29** [comment]
Invalid Path in liveServer.settings.root settings. live Server will serve from workspace root facing this issue i have updated this setting with "/" default root. But not able to Go live option
